src/resources/items/ItemHistory.tsx

```diff
@@ -0,0 +1,25 @@
+import { Datagrid, DateField, List, TextField } from 'react-admin'
+import { useParams } from 'react-router-dom'
+import SourceField from '../../components/SourceField'
+import * as constants from '../../constants'
+import FieldSet from '../../components/FieldSet'
+
+export default function ItemHistory(): React.ReactElement {
+  const { id } = useParams()
+  return (
+    <FieldSet title='History'>
+      <List
+        resource={constants.R_AUDIT}
+        filter={{
```

Comment:
@salony75way -  I think we also need to filter for entries with `resource: R_ITEMS`.  The id `1` could refer to audit entries for a range of resource types

Reply:
Aah, one other filter, please. Could we exclude items where the activityType is `AuditType.EDIT`?

The full list of edits would be available under `History`, but not in this summary view.